pkg/operator/operator.go

```diff
@@ -435,3 +455,40 @@ func (optr *Operator) sync(key string) error {
 
 	return optr.syncAll(syncFuncs)
 }
+
+func (optr *Operator) eventHandlerSingleton(f func(interface{}) bool) cache.FilteringResourceEventHandler {
+	workQueueKey := fmt.Sprintf("%s/%s", optr.namespace, optr.name)
+	addToQueue := func(obj interface{}) {
+		logResource(obj)
+		optr.queue.Add(workQueueKey)
+	}
+
+	return cache.FilteringResourceEventHandler{
+		FilterFunc: f,
+		Handler: cache.ResourceEventHandlerFuncs{
+			AddFunc:    addToQueue,
+			DeleteFunc: addToQueue,
+			UpdateFunc: func(old, new interface{}) {
+				addToQueue(new)
+			},
+		},
+	}
+}
+
+func logResource(obj interface{}) {
+	metaObj, okObject := obj.(metav1.Object)
+	if !okObject {
+		klog.Errorf("Error assigning type to interface when logging")
+	}
+	klog.V(4).Infof("Resource type: %T", obj)
```

Comment:
I've often found just a `klog.Infof` works best if you actually want to see this by default, otherwise I believe this requires a higher debugging log level?

Reply:
its just debug so v4 is fine